src/Kerbero.Common/Repositories/IAuthenticationRepository.cs

```diff
@@ -0,0 +1,15 @@
+using Kerbero.Common.Entities;
+
+namespace Kerbero.Common.Repositories;
+
+public interface IAuthenticationRepository
+{
+	SmartLockNukiAccount AddSmartLockAccount(SmartLockNukiAccount smartLockNukiAccount);
+
+	/// <summary>
+	/// Writes changes in the database
+	/// </summary>
+	/// <param name="cancellationToken"></param>
+	/// <returns>the number of entries written in the database</returns> 
+	public abstract Task<int> SaveChangesAsync(CancellationToken cancellationToken = default);
```

Comment:
xk?

Reply:
It is an entity framework method. Is it better put it directly inside the other methods?